doc/level1.rst

```diff
@@ -427,18 +428,18 @@ Recommendations for single-recipient messages
 
 The Autocrypt recommendation for a message composed to a single
 recipient with the e-mail address ``A`` depends primarily on the value
-stored in :ref:`peer_state[A] <peer-state>`. It is derived
+stored in :ref:`peers[A] <peers>`. It is derived
 by the following algorithm:
 
-1. If there is no peer state, the recommendation is ``disable``.
+1. If there is no ``peers`` entry, the recommendation is ``disable``.
 2. If there is no ``public_key``, the recommendation is ``disable``.
 3. If the ``public_key`` is known for some reason to be unusable for
    encryption (e.g., it is otherwise known to be revoked or expired),
    then the recommendation is ``disable``.
 4. If the message is composed as a reply to an encrypted message, then
    the recommendation is ``encrypt``.
 5. If both ``state`` is ``mutual`` and
-   ``own_state.prefer_encrypt`` is ``mutual``, then the
+   ``accounts[A].prefer_encrypt`` is ``mutual``, then the
```

Comment:
The ``A`` in ``accounts[A]`` is a different ``A`` than the one in ``peers[A]``.  Not sure of the best way to avoid this confusion.  Maybe always use ``peers[P]`` and ``accounts[A]`` ?